pandas/io/excel.py

```diff
@@ -269,6 +287,14 @@ def _get_default_writer(ext):
     return _default_writers[ext]
 
 
+def _is_int_or_intlist(usecols):
```

Comment:
what is this for?

Reply:
It's a function I wrote to check if a function must parse inside `_read_excel` function the columns in `usecols` when `usecols` is a list of ints or a int indicating the fist n columns to be parsed. I did this way because if `usecols` if one of those I have to make it None in line 700 after it's used because if I don't the `index_col` named argument causes a bug after passed in TextParser class because it behaves differently the comparing to its behavior inside `read_excel` function.